### PR TITLE
[new release] minicaml (0.1)

### PR DESCRIPTION
**Pull request by 0x0f0f0f**
A simple, didactical, purely functional programming language

- Project page: <a href="https://github.com/0x0f0f0f/minicaml">https://github.com/0x0f0f0f/minicaml</a>
- Documentation: <a href="https://github.com/0x0f0f0f/minicaml">https://github.com/0x0f0f0f/minicaml</a>

### Added
- Initial release of the project

### CODE REVIEW

File: packages/minicaml/minicaml.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+dev-repo: "git+https://github.com/0x0f0f0f/minicaml.git"
+license: "MIT"
+synopsis: "A simple, didactical, purely functional programming language"
+description: "A simple, didactical, purely functional programming language written for the programming 2 course at the University of Pisa, extended with a simple parser made with Menhir and ocamllex"
+doc: "https://github.com/0x0f0f0f/minicaml"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+    "dune"
```
Comment (kit-ty-kate) [inline packages/minicaml/minicaml.0.1/opam]:
```suggestion
    "dune" {>= "2.0"}
```
According to your `dune-project file, this packages requires at least dune 2.0

**Comment by kit-ty-kate** [comment]
Apparently the archive doesn't exist yet:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - minicaml.0.1:
            https://github.com/0x0f0f0f/minicaml/releases/download/0.1/minicaml-0.1.tbz
            (curl: code 404 while downloading https://github.com/0x0f0f0f/minicaml/releases/download/0.1/minicaml-0.1.tbz)
```

**Comment by 0x0f0f0f** [comment]
Fixed, 

> Apparently the archive doesn't exist yet:
> 
> ```
> [ERROR] The sources of the following couldn't be obtained, aborting:
> - minicaml.0.1:
>   https://github.com/0x0f0f0f/minicaml/releases/download/0.1/minicaml-0.1.tbz
>     (curl: code 404 while downloading https://github.com/0x0f0f0f/minicaml/releases/download/0.1/minicaml-0.1.tbz)
> ```

Solved. Forgot to publish release yesterday.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**